about.html

```diff
@@ -0,0 +1,173 @@
+        contested by senior men's national teams.The current format involves a
+        qualification phase, which takes place over the preceding three years,
+        to determine which teams qualify for the tournament phase. In the
+        tournament phase, 32 teams, including the automatically qualifying host
+        nation(s), compete for the title at venues within the host nation(s)
+        over about a month.As of the 2018 FIFA World Cup, twenty-one final
+        tournaments have been held and a total of 79 national teams have
+        competed. The trophy has been won by eight national teams. Brazil have
+        won five times, and they are the only team to have played in every
+        tournament.France is the defending champion who won it in 2018. This
+        year's world cup will be held in Qatar from 22nd Nobvember to 21st
+        December.
+      </p>
+    </header>
+    <main>
+      <section class="logo-section">
+        <h2 class="sec-title">
+          Dont Miss Your Favorite Players
+          <span class="indicator"></span>
+        </h2>
+
+        <div class="logo-demo">
+          <img src="images/AMC-logos/cr7.png" alt="amc-white-logo" />
+          <img src="images/AMC-logos/messi.png" alt="amc-white-logo" />
+          <img src="images/AMC-logos/pogba.png" alt="amc-white-logo" />
+          <img src="images/AMC-logos/rudiger.jpg" alt="amc-white-logo" />
+        </div>
+      </section>
+      <section class="past-event-section" id="event">
+        <h2 class="sec-title">
+          see previous opening ceremonies for World Cup
+          <span class="indicator"></span>
+        </h2>
+        <p>
+          Take a look at the last two Opening ceremonies which took place in
+          Brasilia and Moscow respectively.
+        </p>
+        <ul class="past-events">
+          <li>
+            <h3>2014 <span>World Cup opening ceremony</span></h3>
+          </li>
+          <li>
+            <h3>2018 <span>World Cup opening ceremony</span></h3>
+          </li>
+        </ul>
+      </section>
+      <section class="partner-section">
+        <h2 class="sec-title">
+          partner
+          <span class="indicator"></span>
+        </h2>
+        <div class="partners-logo-container">
+          <img src="images/Partners/cocacola.png" alt="cocacola-logo" />
+          <img src="images/Partners/adidas.png" alt="adidas-logo" />
+          <img src="images/Partners/hp.png" alt="hp-logo" />
+          <img src="images/Partners/MTN-Logo.png" alt="mtn-logo" />
+          <img
+            src="images/Partners/standard-chartered.png"
+            alt="standard-chartered-logo"
+          />
+        </div>
+      </section>
+    </main>
+    <footer>
+      <div class="footer-logo">
+        <img src="images/AMC-logos/Qatar-logo.jpg" alt="worldcup-logo" />
+      </div>
+      <p class="footnote">
+        2022 World cup Qatar
+        <span>&copy; Some rights reserved. </span>
+      </p>
+    </footer>
```

Comment:
- [OPTIONAL] Your logo on the 'About' page is larger than the footer, making it look strange. I would suggest you shrink the logo size a little so it can fit in the footer. There is also the same alignment issue that the 'Home' page footer has, I would try to make it look consistent on both pages, using the same classes to style them and only changing the background color in the desktop view.